indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -218,9 +231,24 @@
 }
 
 .abstract-comment-badges {
-    align-self: flex-end;
+    margin-right: auto;
+    align-self: baseline;
     color: $gray;
-    margin-left: 1em;
+}
+
+.i-timeline-item-metadata {
+    .abstract-metadata-context {
+        align-self: baseline;
+        margin-left: auto;
```

Comment:
In this case you probably to apply `flex-grow: 1` to `.abstract-metadata-action` instead.